api/src/main/java/com/valtech/aem/saas/api/caconfig/SearchCAConfigurationModel.java

```diff
@@ -52,4 +52,25 @@
      * @return true if enabled.
      */
     boolean isAutoSuggestEnabled();
+
+    /**
+     * Checks whether autocomplete feature is enabled.
+     *
+     * @return true if enabled.
+     */
+    boolean isAutocompleteEnabled();
+
+    /**
+     * Gets the autocomplete init threshold.
```

Comment:
Could you mention that this is minimum no of chars to be entered inside of the javadoc?